src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java

```diff
@@ -48,13 +48,14 @@ public Document generateC7DraftDocument(
         @PathVariable("caseId") String caseId,
         @PathVariable("partyId") String partyId,
         @RequestHeader(HttpHeaders.AUTHORIZATION) @Parameter(hidden = true) String authorisation,
+        @RequestHeader("isWelsh") boolean isWelsh,
```

Comment:
Why this as a parameter? We need to generate welsh only if there is a welsh check at the case level

Reply:
Understood and agreed